### DISCUSSION TITLE
63

### DISCUSSION THREAD

**Discussion by giscus** [Shane-Park/markdownBlog#185] [Comments]
# 63

중간프로젝트에서 네이버, 카카오, 구글등의 OAuth 로그인을 구현하고 싶었지만 시간에 치여 불가능 했었는데요, 이번에 조별 과제를 통해 기회가 생겨 파이썬에서 구현 해 보았습니다. developers.google.com/identity/sign-in/web/sign-in Integrating Google Sign-In into your web app Google Sign-In manages the OAuth 2.0 flow and token lifecycle, simplifying your integration with Google APIs. A user always has the option to revoke access to an application at any time. This document..

https://shanepark.tistory.com/63?restoreScroll=19274.5

**Comment by legun078** [comment]
googleLogin.db 와 oogleLogin.user 은 어디서 나온건가요?
